src/test/java/io/openshift/booster/HttpApplicationStandaloneIT.java

```diff
@@ -29,7 +29,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 @RunWith(VertxUnitRunner.class)
-public class HttpApplicationTest {
+public class HttpApplicationStandaloneIT {
```

Comment:
In order to explicitly separate "standalone" tests and OpenShift tests. Now the "standalone" tests run first and then **just** OpenShift tests are executed.